lib/tasks_private/spec_helper.rake

```diff
@@ -0,0 +1,157 @@
+namespace :manageiq do
+  namespace :providers do
+    namespace :ansible_tower do
```

Comment:
I dont think this should be nested under `manageiq:providers`
This task is only visible inside the ansible tower gem.
Maybe `spec:populate_tower` ? because it's spec related?